project-words.txt

```diff
@@ -155,6 +155,7 @@ Knapen
 Koyeb
 Koyeb's
 Lamana
+lastupdated
```

Comment:
if this word appeared here, it's likely because there's a lowercase `lastupdated` instead of `lastUpdated` somewhere in your code, docs or comments. Maybe I'm wrong, but worth double checking.